nurseconnect/forms.py

```diff
@@ -180,22 +203,10 @@ def __init__(self, *args, **kwargs):
         self.set_initial(self.user)
 
     def clean_username(self):
-        username = self.cleaned_data["username"]
-        if username:
-            raw_username = username.raw_input
-            if raw_username[0] == "0":
-                self.cleaned_data["username"] = \
-                    INT_PREFIX + raw_username[1:len(username)]
-            else:
-                self.cleaned_data["username"] = raw_username
-
-        if self.user.username != self.cleaned_data["username"]:
-            if User.objects.filter(
-                username__iexact=self.cleaned_data["username"]
-            ).exists():
-                self._errors["username"] = self.error_class(
-                    ["Username already exists."]
-                )
+        if User.objects.filter(
```

Comment:
Does the current implementation of PhoneNumberField normalize the phonenumber input to +27?